File: Python/백준/Bronze/2501. 약수 구하기/약수 구하기.py
n, k = map(int, input().split())

a_list = []

for i in range(1, n+1):
    if n%i == 0:
        a_list.append(i)

if len(a_list) < k:
    print(0)
else:
    print(a_list[k-1])

File: 백준/Bronze/2501. 약수 구하기/약수 구하기.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split(" ");

const n = parseInt(input[0]); // 첫 번째 값: n
const k = parseInt(input[1]); // 두 번째 값: k

let arr = [];

for (let i = 1; i <= n; i++) {
    if (n % i === 0) {
        arr.push(i);
    }
}

console.log(arr[k - 1] || 0);